src/java/arjdbc/util/DateTimeUtils.java

```diff
@@ -577,4 +590,102 @@ private static int extractIntValue(final CharSequence str, int beg, int end) {
         return n;
     }
 
+
+    private static final char[] ZEROS = {'0', '0', '0', '0', '0', '0'};
+    private static final char[][] NUMBERS;
+
+    static {
+        // maximum value is 60 (seconds)
+        NUMBERS = new char[60][];
+        for (int i = 0; i < NUMBERS.length; i++) {
+            NUMBERS[i] = ((i < 10 ? "0" : "") + Integer.toString(i)).toCharArray();
+        }
+    }
+
+    public static String timestampTimeToString(final ThreadContext context,
+                                               final IRubyObject value, DateTimeZone zone, boolean withZone) {
+        if (value instanceof RubyFloat) {
+            final double dv = ((RubyFloat) value).getValue();
+            if (dv == Double.POSITIVE_INFINITY) {
+                return "infinity";
```

Comment:
what I do not like is that this piece of utility is now postgres specific.
we shall at least mention that as the method's javadoc so it does not get 'unexpectedly' used elsewhere.

Reply:
@dr-itz yeah I agree with @kares.  Maybe even move this into postgresql packaged class as static utility methods.  We have been trying to make sure any special logic does not get confused and used by another adapter.  Even if it works then at a later point we won't know if any logic is specific to one thing or all things.

Reply:
agreed, the infinity stuff doesn't belong here; will move to PG specific code